vapi/libmutter.vapi

```diff
@@ -793,6 +793,7 @@ namespace Meta {
 	public sealed class SelectionSourceMemory : Meta.SelectionSource {
 		[CCode (has_construct_function = false, type = "MetaSelectionSource*")]
 		public SelectionSourceMemory (string mimetype, GLib.Bytes content);
```

Comment:
AFAIK you can just do something like this:
```suggestion
		public SelectionSourceMemory (string mimetype, GLib.Bytes content) throws GLib.Error;
```
which avoids having to use the static function

Reply:
It didn't work for me when I tried it, but now for some reason it just works